src/components/FeatureNotification/UserNotification.tsx

```diff
@@ -37,9 +40,18 @@ export default function UserNotification(): Notification | null {
     };
 
     const notifyTimeZoneUpdates = (userTz: InitializedTimeZone) => {
-      const notifyUser = userTz.timeZone && !userTz.timeZoneAcknowledgedOnMs;
+      const notifyUser =
+        userTz.timeZone &&
+        userTz.timeZoneNotificationCreatedMs &&
+        DateTime.now().plus({ days: -30 }).toMillis() <= userTz.timeZoneNotificationCreatedMs;
```

Comment:
Does this show the notification for 30 days even if the user has acknowledged it?

Reply:
yes, that's the plan (similar to other events).. if acknowledged, we show the notification as 'read'